src/components/pages/lessons/overlay/go-pro-cta-overlay.tsx

```diff
@@ -280,4 +288,22 @@ const GoProCtaOverlay: FunctionComponent<JoinCTAProps> = ({lesson}) => {
   )
 }
 
-export default GoProCtaOverlay
+const OverlayParent: FunctionComponent<JoinCTAProps> = ({lesson}) => {
+  const {query, reload} = useRouter()
+
+  const {session_id} = query
+
+  if (session_id) {
+    return (
+      <ConfirmMembership
+        lesson={{lesson}}
```

Comment:
I think this was turning into `{{lesson: lesson}}` and instead needs to be `{lesson}`.

```suggestion
        lesson={lesson}
```